jwst/superbias/bias_sub.py

```diff
@@ -12,25 +10,20 @@
 
 def do_correction(input_model, bias_model):
     """
-    Short Summary
-    -------------
-    Execute all tasks for Super-Bias Subtraction
+    Execute all tasks for Super-Bias Subtraction.
 
     Parameters
     ----------
-    input_model: data model object
-        science data to be corrected
-
-    bias_model: super-bias model object
-        bias data
+    input_model : data model object
```

Comment:
Here and elsewhere, it might be good to be more specific about which types of JWSTDataModels are acceptable, and use specific class names (i.e., what would come out of `type(input_model)`)

I assume the bias model needs to be specifically a `SuperBiasModel`, for example.